Neos.Utility.Lock/Classes/TYPO3/Flow/Utility/Lock/Lock.php

```diff
@@ -78,6 +77,18 @@ public static function setLockManager(LockManager $lockManager = null)
 
     /**
      * Releases the lock
+     * @param boolean $exclusiveLock TRUE to, acquire an exclusive (write) lock, FALSE for a shared (read) lock. An exclusive lock is the default.
+     * @return void
+     */
+    public function acquire($exclusiveLock)
```

Comment:
Basically removing Lock::acquire method is the best solution. When you instanciate a Lock, it acquire the lock, you can release. But if you need a new Lock, you need to instanciate a new object